packages/website-frontend/src/pages/report.page.ts

```diff
@@ -0,0 +1,63 @@
+import { LitElement, html } from 'lit'
+import { customElement, state } from 'lit/decorators.js'
+
+import 'mutation-testing-elements';
+
+import { reportService } from '../services/report.service';
+import { Report } from '@stryker-mutator/dashboard-common';
+
+@customElement('stryker-dashboard-report-page')
+export class ReportPage extends LitElement {
+  @state()
+  didNotFindReport = false;
+
+  @state()
+  report: Report | undefined;
+
+  @state()
+  sse: string | undefined;
+
+
+  override firstUpdated(): void {
+    // get slug from url
+    const slug = window.location.pathname.split('/reports/').join('');
+    reportService.getReport(slug)
+      .then((report) => {
+        if (report) {
+          document.body.style.backgroundColor = 'rgb(24, 24, 27)';
+          this.report = report;
+        } else {
+          this.didNotFindReport = true;
+        }
+      });
+  }
+
+  override render() {
+    if (this.didNotFindReport) {
+      return html `
+        <sme-spatious-layout>
+          <sme-notify message="Report could not be found..."/>
+        </sme-spatious-layout>
+      `;
+    }
+
+    return html `
+      <mutation-test-report-app .report="${this.report}" @theme-changed=${this.#handleThemeChange}></mutation-test-report-app>
+    `;
+  }
+
+  #handleThemeChange(event: CustomEvent): void {
+    if (event.detail.theme === 'light') {
+      document.body.style.backgroundColor = 'white';
+      return;
```

Comment:
The themeChange event also has a `themeBackgroundColor` property you can use:

```js
document.body.style.backgroundColor = event.detail.themeBackgroundColor
```

Reply:
We could export the event types in mutation-testing-elements so this method can be properly typed